packages/react-data-grid/src/common/cells/headerCells/FilterableHeaderCell.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { useState } from 'react';
+import { Column } from '../../types';
+
+interface Props {
+  column: Column;
+  onChange(change: { filterTerm: string; column: Column }): void;
+}
+
+export default function FilterableHeaderCell({ column, onChange }: Props) {
+  const [filterTerm, setFilterTerm] = useState('');
+
+  function handleChange(event: React.ChangeEvent<HTMLInputElement>) {
```

Comment:
is there any reason you prefer named function vs arrow function?

Reply:
Mostly taste. They're shorter, quicker to parse for humans, easier to type as well.

They're also hoisted, so you can't run into any mutual recursion bugs. You could even place them after the `return` and they'd still work.
![image](https://user-images.githubusercontent.com/567105/55737643-7da49400-5a1d-11e9-8c4d-8fc70c6d9e91.png)